app/models/checks/accessibility_page_heading.rb

```diff
@@ -1,7 +1,7 @@
 module Checks
   class AccessibilityPageHeading < Check
     PRIORITY = 22
-    REQUIREMENTS = [:find_accessibility_page]
+    REQUIREMENTS = Check::REQUIREMENTS + [:find_accessibility_page]
```

Comment:
Bien vu 👍 
Il serait peut-être plus explicite de mettre 
```suggestion
    REQUIREMENTS = [:reachable, :find_accessibility_page]
```

Reply:
Je préfère qu'on laisse le côté dynamique justement pour éviter cette erreur à nouveau